### DISCUSSION TITLE
Upgrading Cocoa Pods

### DISCUSSION THREAD

**Discussion by icarusoft** [codemagic-ci-cd/codemagic-docs#1254] [Q&A] [answered]
I have a iOS build that just started failing after a upgrade of one of the dependencies with this message:

` CDN: trunk Redirecting from https://cdn.cocoapods.org/Specs/4/2/c/FlutterMacOS/3.0.0/FlutterMacOS.podspec.json to https://cdn.jsdelivr.net/cocoa/Specs/4/2/c/FlutterMacOS/3.0.0/FlutterMacOS.podspec.json
      CDN: trunk Relative path downloaded: Specs/4/2/c/FlutterMacOS/1.12.1303/FlutterMacOS.podspec.json, save ETag: W/"a2c-3l7f6SLgY2HFklEJBcPSy7vCMWQ"
      CDN: trunk Relative path downloaded: Specs/4/2/c/FlutterMacOS/1.9.102/FlutterMacOS.podspec.json, save ETag: W/"a2a-fs/UyQIY577K/3M8Ewg1MvspJCA"
      CDN: trunk Relative path downloaded: Specs/4/2/c/FlutterMacOS/2.0.0/FlutterMacOS.podspec.json, save ETag: W/"a30-LxtFbE2UNsTU0TkZvTvo7wO6YyI"
      CDN: trunk Relative path downloaded: Specs/4/2/c/FlutterMacOS/1.22.4/FlutterMacOS.podspec.json, save ETag: W/"a29-rCEO9xlEs/BHiQdtcNTHLH7Axhk"
      CDN: trunk Relative path downloaded: Specs/4/2/c/FlutterMacOS/1.17.0/FlutterMacOS.podspec.json, save ETag: W/"a06-kUX4eP4wEg0WifAqbw0mvmQak/c"
      CDN: trunk Relative path downloaded: Specs/4/2/c/FlutterMacOS/2.10.2/FlutterMacOS.podspec.json, save ETag: W/"9f0-1oaoZn36NKQE7v1b/MUE20unGtQ"
      CDN: trunk Relative path downloaded: Specs/4/2/c/FlutterMacOS/2.2.2/FlutterMacOS.podspec.json, save ETag: W/"a3c-0iXfw7ucDo+DNkugXnApNY3JP+0"
      CDN: trunk Relative path downloaded: Specs/4/2/c/FlutterMacOS/3.0.0/FlutterMacOS.podspec.json, save ETag: W/"9ef-ReUIyH4BioQ2hgfBIDtKWWKawnk"
      CDN: trunk Relative path downloaded: Specs/4/2/c/FlutterMacOS/2.5/FlutterMacOS.podspec.json, save ETag: W/"a3a-B4rs1ilNC3vOEcbc3cs+eBhWegQ"
      CDN: trunk Relative path: Specs/4/2/c/FlutterMacOS/3.0.0/FlutterMacOS.podspec.json modified during this run! Returning local
    [!] CocoaPods could not find compatible versions for pod "ObjectBox":
      In snapshot (Podfile.lock):
        ObjectBox (= 1.7.0)

      In Podfile:
        objectbox_flutter_libs (from `.symlinks/plugins/objectbox_flutter_libs/ios`) was resolved to 0.0.1, which depends on
          ObjectBox (= 1.8.1-rc)

    You have either:
     * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
     * changed the constraints of dependency `ObjectBox` inside your development pod `objectbox_flutter_libs`.
       You should run `pod update ObjectBox` to apply changes you've made.`

Running the "pod repo update" and "pod update ObjectBox" locally seems to fix the issue, but how do I fix this on Codemagic?

Best regards
Søren

**Comment by K-Codemagic** [comment, marked as answer]
You can run the same commands in Codemagic, to resolve the issue. If you are using the Workflow editor you can click on the + sign and add it to the post-clone script section.
